components/EventsMap.tsx

```diff
@@ -94,12 +88,11 @@ export type MapProps<T extends MapMarker> = {
 /**
  * A view for rendering a google map.
  */
-export const Map = forwardRef(function ReffedMap<T extends MapMarker> (
+export const EventsMap = forwardRef(function ReffedMap<T extends MapMarker> (
   {
     initialRegion,
     markers,
     renderMarker,
```

Comment:
I wonder if we'll still need renderMarker too, or if we'll use the profile pic of each event similar to the map snippet